pkg/simulation/passenger/passenger_store.go

```diff
@@ -81,6 +84,126 @@ func (ps *PassengersStore) generatePassengers(c *city.City) {
 	}
 }
 
+func (ps *PassengersStore) GeneratePassengersDueModel(c *city.City, passengerModel []byte) error {
+	reader := csv.NewReader(bytes.NewReader(passengerModel))
+	reader.Comma = ','
+
+	records, err := reader.ReadAll()
+	if err != nil {
+		return fmt.Errorf("error reading passenger model csv: %w", err)
+	}
+
+	if len(records) == 0 {
+		return fmt.Errorf("passenger model csv is empty")
+	}
+
+	header := records[0]
+	if len(header) < 4 {
+		return fmt.Errorf("invalid header, expected at least 4 columns, got %d", len(header))
+	}
+
+	stopsByName := c.GetStopsByName()
+	resolveStopsID := func(stopName string) ([]uint64, error) {
+		if stopName == "" {
+			return nil, fmt.Errorf("empty stop group name")
+		}
+
+		if group, ok := stopsByName[stopName]; !ok || len(group) == 0 {
+			return nil, fmt.Errorf("stop group %q not found", stopName)
+		} else {
+			ids := make([]uint64, 0, len(group))
+			for id := range group {
+				ids = append(ids, id)
+			}
+			return ids, nil
+		}
+	}
+
+	ps.mu.Lock()
+	defer ps.mu.Unlock()
+
+	ps.passengersToSpawn = make(map[uint][]passengerSpawn)
+
+	var counter uint64
+
+	for i, row := range records[1:] {
+		lineNo := i + 2
+
+		if len(row) < 4 {
+			return fmt.Errorf("line %d: expected 4 columns, got %d", lineNo, len(row))
+		}
+
+		startName := strings.TrimSpace(row[0])
+		endName := strings.TrimSpace(row[1])
+		spawnTimeStr := strings.TrimSpace(row[2])
+		strategyStr := strings.TrimSpace(row[3])
+
+		startStopsID, err := resolveStopsID(startName)
+		if err != nil {
+			return fmt.Errorf("line %d: %w", lineNo, err)
+		}
+
+		endStopsID, err := resolveStopsID(endName)
+		if err != nil {
+			return fmt.Errorf("line %d: %w", lineNo, err)
+		}
+
+		var spawnSeconds uint
+		if t, err := time.Parse("15:04:05", spawnTimeStr); err == nil {
+			spawnSeconds = uint(t.Hour()*3600 + t.Minute()*60 + t.Second())
+		} else {
+			return fmt.Errorf("line %d: invalid spawn_time %q (expected HH:MM:SS)", lineNo, spawnTimeStr)
+		}
+
+		strategy := travelplan.PassengerStrategy(strings.ToUpper(strategyStr))
+		switch strategy {
+		case travelplan.RANDOM:
+		case travelplan.ASAP, travelplan.COMFORT, travelplan.SURE:
+			strategy = travelplan.RANDOM
```

Comment:
Could you add a TODO comment here, so we do not forget to remove this assignment when strategies get implemented?